cmake/options.cmake

```diff
@@ -84,6 +84,12 @@ if (NOT WITH_VIRTIO_DEVICE AND NOT WITH_VIRTIO_SLAVE)
 endif (NOT WITH_VIRTIO_DEVICE AND NOT WITH_VIRTIO_SLAVE)
 
 option (WITH_VIRTIO_MMIO "Build with virtio mmio (front end) enabled" ON)
+option (WITH_HVL_VIRTIO "Build with hypervisor-less virtio (front end) enabled" ON)
```

Comment:
Please could you send commit related to the hypervisor less virtio in a separate PR.
To be able to understand your commit I would need also more details about the rational of your update